CHANGELOG.md

```diff
@@ -4,6 +4,301 @@ This changelog goes through all the changes that have been made in each release
+  * Abort merge if head has been updated before pressing merge (#18032)
+  * Improve TestPatch to use git read-tree -m and implement git-merge-one-file functionality (#18004)
+  * Use JSON module instead of stdlib json (#18003)
+  * Fixed issue merged/closed wording (#17973)
+  * Return nicer error for ForcePrivate (#17971)
+  * Fix overflow in commit graph (#17947)
+  * Prevent services/mailer/mailer_test.go tests from deleteing data directory (#17941)
+  * Use disable_form_autofill on Codebase and Gitbucket (#17936)
+  * Fix a panic in NotifyCreateIssueComment (caused by string truncation) (#17928)
+  * Fix markdown URL parsing (#17924)
+  * Apply CSS Variables to all message elements (#17920)
+  * Improve checkBranchName (#17901)
+  * Update chi/middleware to chi/v5/middleware (#17888)
+  * Fix position of label color picker colors (#17866)
+  * Fix ListUnadoptedRepositories incorrect total count (#17865)
+  * Remove whitespace inside rendered code `<td>` (#17859)
+  * Make Co-committed-by and co-authored-by trailers optional (#17848)
+  * Fix value of User.IsRestricted when oauth2 user registration (#17839)
+  * Use new OneDev /milestones endpoint (#17782)
+  * Prevent deadlock in TestPersistableChannelQueue (#17717)
+  * Simplify code for writing SHA to name-rev (#17696)
+  * Fix database deadlock when update issue labels (#17649)
+  * Add warning for BIDI characters in page renders and in diffs (#17562)
+  * Fix ipv6 parsing for builtin ssh server (#17561)
+  * Multiple Escaping Improvements (#17551)
+  * Fixes #16559 - Do not trim leading spaces for tab delimited (#17442)
+  * Show client-side error if wiki page is empty (#17415)
+  * Fix context popup error (#17398)
+  * Stop sanitizing full name in API (#17396)
+  * Fix issue close/comment buttons on mobile (#17317)
+  * Fix navbar UI (#17235)
+  * Fix problem when database id is not increment as expected (#17229)
+  * Open the DingTalk link in browser (#17084)
+  * Remove heads pointing to missing old refs (#17076)
+  * Fix commit status index problem (#17061)
+  * Handle broken references in mirror sync (#17013)
+  * Fix for create repo page layout (#17012)
+  * Improve LDAP synchronization efficiency (#16994)
+  * Add repo_id for attachment (#16958)
+  * Clean-up HookPreReceive and restore functionality for pushing non-standard refs (#16705)
+  * Remove duplicate csv import in modules/csv/csv.go (#16631)
+  * Improve SMTP authentication and Fix user creation bugs  (#16612)
+  * Fixed emoji alias not parsed in links (#16221)
+  * Calculate label URL on API  (#16186)
+* TRANSLATION
+  * Fix mispelling of starred as stared (#17465)
+  * Re-separate the color translation strings (#17390)
+  * Enable Malayalam, Greek, Persian, Hungarian & Indonesian by default (#16998)
+* BUILD
+  * Add lockfile-check (#18285)
+  * Don't store assets modified time into generated files (#18193)
+  * Use shadowing script for docker (#17846)
+* MISC
+  * Update JS dependencies (#17611)
```

Comment:
```suggestion
  * Update JS dependencies (#17611 and others)
```

Reply:
I would not add this if you have other pulls, we should add there number else just let it as is